package.json

```diff
@@ -73,7 +73,7 @@
     "@elastic/elasticsearch-v7.17.0": "npm:@elastic/elasticsearch@7.17.0",
     "@elastic/elasticsearch-v7.9.0": "npm:@elastic/elasticsearch@7.9.0",
     "@google-cloud/pubsub": "4.0.2",
-    "@google-cloud/storage": "^6.9.4",
+    "@google-cloud/storage": "^7.5.0",
```

Comment:
_issue:_ IIRC version 7 dropped support for Node.js 12, making Node.js 14 the minimum required version. That means we need to update for which versions we execute the test. Currently we skip the test when the current Node.js version is < 12. See https://github.com/instana/nodejs/blob/main/packages/collector/test/tracing/cloud/gcp/storage/test.js#L56-L64